syntax = "proto3"; // PB协议版本
package aiins;

option java_package = "com.example.aiins.proto";

// 消息获取请求
message MessageReq {
    int32 uid = 1;
    int32 time = 2;
}

// 消息获取响应
message MessageRsp {
    repeated Message msgs = 1;
}

// 每一个对话消息
message Message {
    int32 src = 1;
    int32 dst = 2;
    int32 time = 3;
    string text = 4;
    bytes image = 5;
}

// 每一个发布的 post
message Post {
    int32 uid = 1;
    string text = 2;
    bytes image = 3;
    int32 time = 4;
}